docs/source/markdown/podman-system-service.1.md

```diff
@@ -13,6 +13,10 @@ If no endpoint is provided, defaults will be used.  The default endpoint for a r
 service is *unix:/run/podman/podman.sock* and rootless is *unix:/$XDG_RUNTIME_DIR/podman/podman.sock* (for
 example *unix:/run/user/1000/podman/podman.sock*)
 
+To access the API service inside a container, you need to
```

Comment:
We try to avoid using pronouns, especially "you" in the man pages.
```suggestion
To access the API service inside a container:
```

Reply:
Done